### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Aericio**
Unofficial, not Inofficial.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ ClearSky is an ultra fast Minecraft: Pocket Edition server software with clean f
  - We may take code from other GPL licenced projects, but most features are fully-rewritten for ClearSky.
  - Here is an Telegram group, where you can communicate with us: [Telegram/ClearSky] (https://telegram.me/joinchat/AlErxAY3tx0MPBGYuGtpDA).
  - [Look here how fast ClearSky starts with 70 plugins and 20 worlds!](http://wolvesfortress.de/ezgif-1446650535.gif)
- - Inofficial Test Server : Creative IP: WolvesFortress.de Port: **19134** *Often offline, but if online, latest version*
+ - Unofficial Test Server : Creative IP: WolvesFortress.de Port: **19134** *Often offline, but if online, latest version*
```
Comment (ghost) [inline README.md:18]: Haha! didnt read that, i should of had fixed it. and Change it to Testing Server? wouldnt it be better

**Comment by robske110** [comment]
INOFFICAL IS RIGHT
Close this

**Comment by inxomnyaa** [comment]
http://wikidiff.com/unofficial/inofficial

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md:18]: nvm

**Comment by Aericio** [comment]
im gay

**Comment by ghost** [comment]
haha

**Pull request closed without merging**